tests/ops/integration_tests/saas/request_override/test_firebase_auth_task.py

```diff
@@ -410,8 +416,8 @@ async def test_firebase_auth_delete_request(
     """
     Tests delete functionality by explicitly invoking the delete override function
 
-    We can't have an'end-to-end' privacy request test, as preferred, because
-    our delete function is not configured by default (the udpate function is).
+    We can't have an 'end-to-end' privacy request test, as preferred, because
+    our delete function is not configured by default (the update function is).
     But this at least gives us some test coverage of the delete function directly.
```

Comment:
It looks like the delete tests aren't running erasure privacy requests because `update` wasn't the default. Now that we've disabled the update tests, can we add the erasure test here?